src/test/javascript/spec/component/assessment-dashboard/exercise-assessment-dashboard.component.spec.ts

```diff
@@ -319,6 +320,9 @@ describe('ExerciseAssessmentDashboardComponent', () => {
                 comp.submissionsWithComplaints = [submissionWithComplaintDTO];
 
                 accountService = TestBed.inject(AccountService);
+                const navigationUitlService = TestBed.inject(ArtemisNavigationUtilService);
+
+                routingStub = jest.spyOn(navigationUitlService, 'routeInNewTab').mockImplementation();
```

Comment:
```suggestion
                const navigationUtilService = TestBed.inject(ArtemisNavigationUtilService);

                routingStub = jest.spyOn(navigationUtilService, 'routeInNewTab');
```

As far as I know, `mockImplementation` shouldn't be needed when `MockProvider` is used.